SyslogNet.Client/Transport/ISyslogMessageSender.cs

```diff
@@ -9,5 +10,8 @@ public interface ISyslogMessageSender : IDisposable
 		void Reconnect();
 		void Send(SyslogMessage message, ISyslogMessageSerializer serializer);
 		void Send(IEnumerable<SyslogMessage> messages, ISyslogMessageSerializer serializer);

```

Comment:
For not breaking current code, yes, they need to be exposed. If someone needs to call a method from the UI thread, it's the `SendAsync` the one they should be invoking, otherwise even with the current implementation, it will block.